helpers/student.js

```diff
@@ -28,14 +29,14 @@ exports.editProfile = async function (req, res, next) {
   catch (err) {
     return res.json({ message: err.message });
   }
-  returnedStudent = await studentModel.findOne({ email: decodedToken.email });
+  returnedStudent = await studentModel.findOne({ _id: decodedToken.id });
   res.send({ returnedStudent });
 }
 
 exports.profile = async function (req, res, next) {
   const token = req.headers.authorization.split(" ")[1];
   const decodedToken = await jwt.verify(token, req.app.get('secretKey'));
-  const student = await studentModel.findOne({ email: decodedToken.email });
+  const student = await studentModel.findOne({ _id: decodedToken.id });
   res.send({ student });
 }
 
```

Comment:
In line 43 (wasn't able to comment directly on it), the findAndUpdate function is defined where the email used to make database requests is passed in. I did not update the function to use id instead because I assumed this was by choice.